about.html

```diff
@@ -110,30 +116,30 @@ <h3 class="port-head">2011</h3>
             <h2 class="headings heading-3">Partners</h2>>
             <div class="main-partners">
                 <a href="">
-                    <img src="img/partner-1.png"  class="single-partner">
+                    <img alt="Robust-Events" src="img/partners/mozilla.png"  class="single-partner">
                 </a>
                 <a href="">
-                    <img src="img/partner-1.png"  class="single-partner">
+                    <img alt="Robust-Events" src="img/partners/google.png"  class="single-partner">
                 </a>
                 <a href="">
-                    <img src="img/partner-1.png"  class="single-partner">
+                    <img alt="Robust-Events" src="img/partners/naver.png"  class="single-partner">
                 </a>
                 <a href="">
-                    <img src="img/partner-1.png"  class="single-partner">
+                    <img alt="Robust-Events" src="img/partners/daumkakao.png"  class="single-partner">
                 </a>
                 <a href="">
-                    <img src="img/partner-1.png"  class="single-partner">
```

Comment:
- [ ] Please, consider modifying the alt name for each img differently: you gave the same name for most imgs I checked - it is called "Robust-Events" 👍